keep/contextmanager/contextmanager.py

```diff
@@ -89,21 +97,24 @@ def set_condition_results(
             compare_value (_type_): _description_
             result (_type_): _description_
             condition_alias (_type_, optional): _description_. Defaults to None.
-            raw_value (_type_): the raw value which the condition was compared to. this is relevant only for foreach conditions
+            value (_type_): the raw value which the condition was compared to. this is relevant only for foreach conditions
```

Comment:
why is a foreach condition different from other condition?

Reply:
its for all of them now, not only for foreacb